gapis/service/service.proto

```diff
@@ -804,6 +804,7 @@ message ErrUnsupportedVersion {
 enum TraceType {
   Graphics = 0;
   Perfetto = 1;
+  Angle = 2;
```

Comment:
Remove this. I know the Type/API combo is a bit confusing as it is right now. I think, to stick with the current non-obvious way, would be to use type Graphics and API "OpenGL on ANGLE", the way you are mostly doing already.

Another option would be to simply make ANGLE a checkbox for a Vulkan trace. What do you think? would that be too confusing to select Vulkan for an OpenGL on ANGLE app?

Reply:
Yep, killed this and couple related functions that aren't needed with how I'm currently detecting ANGLE.

I personally like having the separate drop-down for "OpenGL on ANGLE" vs. a potential checkbox. Helps to clearly distinguish between an OpenGL app and Vulkan app, and also avoids users trying to check the ANGLE box for a Vulkan app.